pkgs/applications/networking/remote/x2goclient/default.nix

```diff
@@ -0,0 +1,31 @@
+{ fetchurl, stdenv, nxproxy, qt4, openldap, libssh, libXpm, cups, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "4.0.3.1";
+  name = "x2goclient-${version}";
+
+  src = fetchurl {
+    url = "http://code.x2go.org/releases/source/x2goclient/x2goclient-4.0.3.1.tar.gz";
+    md5 = "ef9a20ef96f7c31cc20ece9ebbf1e007";
+  };
+
+
+  buildInputs = [ nxproxy qt4 openldap libssh libXpm cups makeWrapper ];
+
+  preBuild = ''
+    lrelease x2goclient.pro;
+    qmake;
+  '';
+  postInstall = ''
+    mkdir -p $out/bin;
+    cp x2goclient $out/bin;
+    cp -a man $out;
+    wrapProgram "$out/bin/x2goclient" --suffix PATH : "${nxproxy}/bin";
+  '';
```

Comment:
Huh. I've never seen semi-colons used like that in shell code before... +1 for personal style? :-)

-1 for readability, I'm afraid.